docs/README.md

```diff
@@ -2957,6 +2958,91 @@ Variables
 
 The databases for the local provider are just json files in the `<project-root>/.booster` folder. If you are wondering what data is available in the application you will only need to chose what file to look into.
 
+### Testing Booster applications
+To properly test a Booster application, you should create a `test` folder at the same level as the `src` one (check the generated `tsconfig.json` for more information). Apart from that, tests' names should have the `<my_test>.test.ts` format.
+
+When a Booster application is generated, you will have a script in a `package.json` like this:
+```typescript
+"scripts": {
+  "test": "nyc --extension .ts mocha --forbid-only \"test/**/*.test.ts\""
+}
+```
+
+The only thing that you should add to this line are the `AWS_SDK_LOAD_CONFIG=true` and `BOOSTER_ENV=test` environment variables, so the script will look like this:
+
+```typescript
+"scripts": {
+  "test": "AWS_SDK_LOAD_CONFIG=true BOOSTER_ENV=test nyc --extension .ts mocha --forbid-only \"test/**/*.test.ts\""
```

Comment:
Why these env vars have to be added manually?

Reply:
Each one solves a different "error":
- BOOSTER_ENV=test => `No environment found for your Booster application`
- AWS_SDK_LOAD_CONFIG=true => `region not found`

I'm not 100% sure why this error happens, but it's just a quick fix @adriangmweb